app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -87,9 +86,14 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
     // work properly, so we !!
     private val youtubeBackHandler by lazy { YouTubeBackHandler(engineView!!, activity as MainActivity) }
 
+    private lateinit var webRenderViewModel: WebRenderViewModel
+    private lateinit var rootView: View
```

Comment:
Let's make sure we remember to null this out in order to prevent leaking.

Reply:
Sure, I'd be surprised if they are not already garbage collected `onDestroy` though (like all the other `lateinit vars`). But to be safe (since we are messing with view references) I will make this nullable and clear it up in `onDestroyView`

Reply:
Unless I'm mistaken, `lateinit vars` don't clean themselves up in `onDestroy`.  Is there some  KTX functionality or something that I'm unaware of?